securedrop/source_templates/lookup.html

```diff
@@ -54,10 +54,10 @@ <h2 id="submit-heading">{{ gettext('Submit Messages') }}</h2>
 
     <div class="form-controls">
       <a href="{{ url_for('main.lookup') }}" class="btn secondary" role="button"
-        aria-label="{{ gettext('Cancel') }}">
+        aria-label="{{ gettext('Cancel and Clear Uploaded Content') }}">
         {{ gettext('CANCEL') }}
       </a>
-      <button type="submit" aria-label="{{ gettext('Submit') }}">
+      <button type="submit" aria-label="{{ gettext('Submit Uploaded Content to the SecureDrop') }}">
```

Comment:
Nit: We tend not to use the definite article before "SecureDrop", in which case:

```suggestion
      <button type="submit" aria-label="{{ gettext('Submit Uploaded Content to SecureDrop') }}">
```

I wonder about emulating [`base.html`'s logic around the `organization_name`](https://github.com/freedomofpress/securedrop/blob/f7ca802b6842e8af812f71cc9dca2800e0089183/securedrop/source_templates/base.html#L8-L12).  But I think this is fine, as would be the even simpler:

```suggestion
      <button type="submit" aria-label="{{ gettext('Submit Uploaded Content') }}">
```